### ISSUE TITLE
The OID_REDIRECT_PATH == http://localhost/<the_value_from_OID_REDIRECT_PATH>

### ISSUE THREAD

**Issue by AdeOpe** [evry/docker-oidc-proxy#5]
Hi,
the OID_REDIRECT_PATH always resolves to http://localhost/<the_value_from_OID_REDIRECT_PATH>

however if you want to resolve to a different url (e.g. when using this with AWS cognito), this fails

**Comment by Starefossen** [comment]
Good catch, @AdeOpe 🙂 Can you test with an absolute path starting with a `/my-redirect-path`?

**Comment by AdeOpe** [comment]
@Starefossen, sorry could you please clarify what you would like me to do?
my callback url in aws cognito is :
https://<dns>/oauth2/idpresponse"

so when i tried set the OID_REDIRECT_PATH to

https://<dns>/oauth2/idpresponse
or
/https://<dns>/oauth2/idpresponse

it gets set to 
http://localhost//https:/<dns>/oauth2/idpresponse
or
http://localhost//https://<dns>/oauth2/idpresponse

**Comment by malagant** [comment]
Any news on this? You seemed to use a complete URL instead of a URI path. I get the same when using a complete URL

**Comment by stehefan** [comment]
As far as I can see from the documentation of [lua-resty-openidc](https://github.com/zmartzone/lua-resty-openidc), the feature to set an absolute URL got introduced in version `1.7.0`. This Dockerfile still uses `1.6.1`, so this feature is not available. See also documentation for [Version 1.7.0](https://github.com/zmartzone/lua-resty-openidc/releases/tag/v1.7.0)
